src/style.css

```
@@ -1 +1,29 @@
-/* styles go here */
+.visually-hidden {
+  position: absolute !important;
+  width: 1px !important;
+  height: 1px !important;
+  margin: -1px !important;
+  border: 0 !important;
+  padding: 0 !important;
+  overflow: hidden !important;
+  clip: rect(0, 0, 0, 0) !important;
+  white-space: nowrap !important;
+}
```

there is no need in these styles. Do not use `!important`. You can read here about it https://uxengineer.com/css-specificity-avoid-important-css/
```suggestion
}
```